src/main/json2xlsx.js

```diff
@@ -4,15 +4,17 @@ const assert = require('assert');
 const fileUtils = require('./lib/file-utils');
 const ccdUtils = require('./lib/ccd-spreadsheet-utils');
 const stringUtils = require('./lib/string-utils');
-const { Substitutor } = require('./lib/substitutor');
+const sheetUtils = require('./lib/sheet-utils');
+const {Substitutor} = require('./lib/substitutor');
```

Comment:
Please add spacing around brackets as it was before.

Reply:
Intellij keeps formatting it that way even though I set the code style to otherwise 😢

Reply:
Intellij auto formatting even though it is told to do otherwise 😢